### PR TITLE
Fixes for particle neutrinos

### PR DESCRIPTION
**Pull request by sbird**
Two trivial fixes for pure particle neutrinos. These have no effect if you are using the linear response neutrinos, which @andreufont is.

### CODE REVIEW

File: genic/main.c [hunk 1]
```diff
@@ -58,13 +58,25 @@ int main(int argc, char **argv)
...
+  /*Massive neutrinos*/
+
+  const int64_t TotNu = (int64_t) NGridNu*NGridNu*NGridNu;
+  double total_nufrac = 0;
+  if(TotNu > 0) {
+    const double kBMNu = 3*CP.ONu.kBtnu / (CP.MNu[0]+CP.MNu[1]+CP.MNu[2]);
+    double v_th = NU_V0(InitTime, kBMNu, UnitVelocity_in_cm_per_s);
+    if(!UsePeculiarVelocity)
+        v_th /= sqrt(InitTime);
+    total_nufrac = init_thermalvel(v_th, Max_nuvel/v_th, 0);
```
Comment (rainwoodman) [inline genic/main.c]: Is this the total fraction of massive neutrino? Would it be useful to save this value in the header as a properly named attr?

File: genic/power.c [hunk 2]
```diff
@@ -92,7 +92,8 @@ void parse_transfer(int i, double k, char * line, struct table *out_tab, int * I
      * Everything is a ratio against tot*/
     out_tab->logD[0][i] = pow(transfers[1]/transfers[5],2);
     out_tab->logD[1][i] = pow(transfers[0]/transfers[5],2);
-    out_tab->logD[2][i] = pow(transfers[6]/transfers[5],2);
+    out_tab->logD[2][i] = pow(transfers[4]/transfers[5],2);
+    out_tab->logD[3][i] = pow(transfers[6]/transfers[5],2);
```
Comment (rainwoodman) [inline genic/power.c]: Could you add a comment which column is which species?

File: genic/main.c (hunk 1 above)
Comment (sbird) [reply genic/main.c]: Yes! This is the fraction of massive neutrino in particles.

File: genic/power.c (hunk 2 above)
Comment (sbird) [reply genic/power.c]: Done

**Comment by sbird** [comment]
Will merge when travis goes green.

**Comment by rainwoodman** [comment]
It is green.

**Pull request merged**